google/cloud/pubsub_v1/futures.py

```diff
@@ -31,156 +30,44 @@ class Future(google.api_core.future.Future):
     methods in this library.
 
     Args:
-        completed (Optional[Any]): An event, with the same interface as
-            :class:`threading.Event`. This is provided so that callers
-            with different concurrency models (e.g. ``threading`` or
-            ``multiprocessing``) can supply an event that is compatible
-            with that model. The ``wait()`` and ``set()`` methods will be
-            used. If this argument is not provided, then a new
-            :class:`threading.Event` will be created and used.
+        completed (Optional[Any]):
+             [deprecated] Not used anymore and will be removed in the future.
```

Comment:
This should be fine, as `Future` objects should not be created by the users directly, and because we already did not really support concurrency model other than threading.

We only have the thread-based implementation of the `Batch` class, and the rest of the code (e.g. `Client.publish()` and `OrderedSequencer.publish()` ) already created `Future` instances using a default `completed`, which was `threading.Event()`.

Reply:
Do we know of any clients that are passing this argument? (I see this package was one. :) )

Do we plan to get rid of this in the near future?